app/controllers/RestController.scala

```diff
@@ -14,55 +15,56 @@ import scala.concurrent.Future
 class RestController @Inject()(
     cc: ControllerComponents,
     silhouette: Silhouette[DefaultEnv],
-    serverCtrl: Controller
+    serverCtrl: Controller,
+    homectrl: HomeController
```

Comment:
I kind of dislike sharing controller instances within each other :/ would be awesome if we could somehow just share a single method.

Thoughts for the future: Maybe the actor-management should be its own class with then could be shared with home and rest controller

Reply:
You're right, I had the same feeling as it came to inject another controller. 
Putting the actor-management in an own class sounds nice, I will create an issue for that but would say it's an 1.x "feature"?

Reply:
yep, definitely 1.x